File: src/main/java/com/battletrident/commands/CommandManager.java
package com.battletrident.commands;

import com.battletrident.commands.command.BT;
import io.papermc.paper.command.brigadier.Commands;
import io.papermc.paper.plugin.lifecycle.event.LifecycleEventManager;
import io.papermc.paper.plugin.lifecycle.event.types.LifecycleEvents;
import org.bukkit.plugin.Plugin;

import static com.battletrident.consts.Plugin.Plugin;

public class CommandManager {
	public CommandManager() {
		final LifecycleEventManager<Plugin> manager = Plugin.getLifecycleManager();

		manager.registerEventHandler(LifecycleEvents.COMMANDS, event -> {
			final Commands commands = event.registrar();
			commands.register(
				"bt",
				"BattleTrident 게임 관리자 패널",
				new BT()
			);
		});
	}
}

File: src/main/kotlin/com/BattleTrident/commands/CommandManager.kt
package com.BattleTrident.commands

import io.papermc.paper.plugin.lifecycle.event.types.LifecycleEvents
import com.BattleTrident.BattleTrident
import com.BattleTrident.commands.command.BT

class CommandManager {
	//private val commands: List<BasicCommand>

	init {
		val lm = BattleTrident.lifecycleManager

		lm.registerEventHandler(LifecycleEvents.COMMANDS) {
			it.registrar().register(
				"bt",
				"BattleTrident 게임 관리자 패널",
				BT()
			)
		}
	}
}